s2nTests/docker/s2n.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:18.04
+
+RUN apt-get update -y -q && \
```

Comment:
let's only keep the packages that are needed, `s2n_install_test_dependencies.sh` installs some of these, including yices and z3